.github/workflows/ubuntu-latest.yml

```diff
@@ -24,8 +24,21 @@ jobs:
       max-parallel: 2
       matrix:
         indexsize: [32, 64]
-        precision: ['double']
-        buildtype: ['Debug', 'Release']
+        # Disable extended tests until compiler warnings are addressed
+        precision: ['single', 'double']
+        buildtype: ['Debug', 'Release', 'RelWithDebInfo']
+        exclude:
+          - buildtype: Debug
+            precision: single
+          # - buildtype: Debug
+          #   precision: extended
+          - buildtype: Release
+            precision: single
+          # - buildtype: Release
+          #   precision: extended
+          - buildtype: RelWithDebInfo
+            precision: double
```

Comment:
Since our "standard" installation uses both `double` and `RelWithDebInfo`, is that really a configuration that we want to "exclude" in this workflow?  I'm guessing that we just exclude some to limit the combinatorial explosion of tests (e.g., once extended precision is running, we'd have 9 possible combinations) -- is there some logic behind this selection of excluded combinations?

Reply:
This reproduces what we currently run for test cases with this workflow. There are other jobs that build `double` without setting the build type so I suspect that's why the original version of the file did not include this case. We probably we need to revisit this can refine our coverage.